apps/music_proxy.cpp

```diff
@@ -221,17 +222,33 @@ class SteeringHandler
         }
     }
 
-    monsteer::steering::SimulationPlaybackState::State getPlaybackState( ) const
+    monsteer::steering::SimulationPlaybackState::State getPlaybackState() const
     {
         return _state;
     }
 
+    std::string deserializeJSON( const zeroeq::FBEvent& event,
```

Comment:
@hernando : This function moved from ZeroEQ to move around JSON for our specific case. The flatbuffers JSON support is removed from ZeroEQ.